export function task() {
    throw new Error('failed task');
}